src/supabase/queries/case.ts

```diff
@@ -0,0 +1,20 @@
+import supabase from '../createClient';
+
+export const getCase = async (caseId: any) => {
+  const { data, error } = await supabase
+    .from('Cases')
+    .select()
+    .eq('id', caseId);
+  return { data, error };
+};
+
+export const uploadCase = async (caseId: any) => {
```

Comment:
Let's move this to `/supabase/queries/cases.ts` and structure it like the other functions in that file!